bHY6yh-d30103/cod-bqe7s7_bHY6yh-d30103.xml

```diff
@@ -0,0 +1,491 @@
+               quidem putamus maximum esse bo<lb ed="#P" break="no"/>norum 
+               ciuitatibus / sic enim utique minime 
+               <pb ed="#P" n="59-v"/>
+               <cb ed="#P" n="a"/>
+               <lb ed="#P"/>seditiones faciet.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e419">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e469">Idem topicorum libro iii</ref> <quote xml:id="bHY6yh-d30103-Qd1e421">Magis eligenda est 
+               <lb ed="#P"/>super abundantia amicorum quam pecuniarum</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e433">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e487">Cronica de nugis philosophorum capitulo xxxiii ¶</ref> <quote xml:id="bHY6yh-d30103-Qd1e437">de 
+               <lb ed="#P"/>aristotele / aristoteles interrogatus quid est 
+               <lb ed="#P"/>amicitia inquit una anima equalens duo 
+               <lb ed="#P"/>corpora 
+               <lb ed="#P"/>Idem amicitiam terminabat esse equili<lb ed="#P" break="no"/>tatem <!--This is a seperate entry-->
+               beniuolentie conuertibilem</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e447">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e503">Iesus filius sirach in ecclesiastico capitulo vi</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d30103-Qd1e451">noli fieri pro amico inimicus / animcus 
+               <lb ed="#P"/>fidelis protectio fortis. Qui autem inuenit 
+               <lb ed="#P"/>illum inuenit thesaurum / Amico fideli 
+               <lb ed="#P"/>nulla est comparatio / et non est digna 
+               <lb ed="#P"/>ponderatio auri et argenti contra boni<lb ed="#P" break="no"/>tatem 
+               fidei eius.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e466">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e522">Ibidem xxv capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e468">Beatus qui inuenit ami<lb ed="#P" break="no"/>cum 
+               verum</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e472">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e482"> 
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e492"> 
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e500">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e568">Salustius in catilinario</ref>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e508"> <!--Combined with above-->
+               <quote xml:id="bHY6yh-d30103-Qd1e510">idem velle atque 
+               <lb ed="#P"/>idem nolle / ea demum firma amicitia 
+               <lb ed="#P"/>est.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e517">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e584">Idem in iugurtino /</ref> <quote xml:id="bHY6yh-d30103-Qd1e521">Non exercitus neque 
```

Comment:
same comment here. The punctuation marks ( slashes or pilcrows) aren't part of the ref or the quotation, so they shouldn't be inside the ref or quote, but should come between the two elements.

```suggestion
               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e584">Idem in iugurtino</ref> / <quote xml:id="bHY6yh-d30103-Qd1e521">Non exercitus neque 
```